modules/virt-add-disk-to-vm.adoc

```diff
@@ -38,6 +38,10 @@ Use this procedure to add a virtual disk to a {object}.
 
 . In the *Add Disk* window, specify the *Source*, *Name*, *Size*, *Type*, *Interface*, and *Storage Class*.
 
+.. Optional: Select the *Enable preallocation* checkbox to improve write performance when creating data volumes. However, this option should only be used if you really need maximum write performance.
```

Comment:
I suggest tightening up the second sentence and making it more specific/informative. What does "if you really need maximum write performance" mean? In general, don't use words like "really" (see the minimalism principles for more info about avoiding unnecessary modifiers). I'd also put this into the active voice.

Suggested rewrite: "Only select this option if you require maximum write performance." However, this still doesn't explain why this is a risky option. Is there a specific risk/tradeoff that you could specify here?